package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "macpro-quickstart-serverless",
-  "type": "module",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "directories": {
     "test": "tests"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "cd tests/cypress && npm test && cd -",
```

Comment:
So I believe for consistency with the rest of the repo using yarn here, please try switching npm here to yarn.  I'm looking at the old unit test driver shell script as an example:
https://github.com/CMSgov/macpro-quickstart-serverless/blob/master/test-unit.sh

Reply:
This applies to line 10 and 11